code/mathematical-algorithms/babylonian_method/babylonian_method.go

```diff
@@ -0,0 +1,21 @@
+package main
```

Comment:
Please add comment  /* Part of Cosmos by OpenGenus Foundation */ at the top of every code you contribute to this project  ! 👍

Reply:
The comment is not a must. Note it 👍